JoinColony/colonyCDapp
# Feat: Add custom transactions action card

## Description

This PR adds the custom transactions card to the manage colony drawer and updates it in the action select dropdown.

Figma file: https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=6111-2669&t=49uR3c8BmTwlpEbS-4

## Testing

* Click "Manage colony" and check Custom transactions appears in the list

<img width="663" alt="Screenshot 2024-12-04 at 17 42 05" src="https://github.com/user-attachments/assets/4c200e68-a502-4edb-af7c-c725d31961e0">

* Click the custom transactions card and check it appears in the action select dropdown under the admin section

<img width="707" alt="Screenshot 2024-12-04 at 17 49 24" src="https://github.com/user-attachments/assets/0e1c2e7f-2f27-46e7-9c84-58f119a2156f">

* Disabled the arbitrary transactions feature flag and make sure it does not appear in either of these lists

## Diffs

**New stuff** ✨

* Custom transactions card added to manage colony group

**Changes** 🏗

* Arbitrary transaction renamed to Custom transactions and moved to admin section

Resolves #3844
